integ-test/src/test/java/org/opensearch/sql/sql/GeoPointIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_GEOPOINT;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+public class GeoPointIT extends SQLIntegTestCase {
+  @Override
+  protected void init() throws Exception {
+    loadIndex(Index.GEOPOINT);
+  }
+
+  @Test
+  public void test_geo_point() throws IOException {
```

Comment:
It appears the you can remove `throws IOException` from all of the tests as it is never thrown